Core/Services/TripService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AutoMapper;
+using Core.DTO.TripDTO;
+using Core.Entities.TripEntity;
+using Core.Exceptions;
+using Core.Interfaces;
+using Core.Interfaces.CustomService;
+using Core.Resources;
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Core.Services
+{
+    public class TripService : ITripService
+    {
+        private readonly IRepository<Trip> _tripRepository;
+        private readonly ICarService _carService;
+        private readonly IMapper _mapper;
+
+        public TripService(
+            IRepository<Trip> tripRepository,
+            ICarService carService,
+            IMapper mapper)
+        {
+            _tripRepository = tripRepository;
+            _carService = carService;
+            _mapper = mapper;
+        }
+
+        public bool CheckIsTripExistsById(int tripId)
+        {
+            return _tripRepository.Query()
+                                  .Any(p => p.Id == tripId);
+        }
+
+        public async Task CreateTripAsync(CreateTripDTO createTripDTO, string creatorId)
+        {
+            ValidateTripDate(createTripDTO.StartDate, createTripDTO.ExpirationDate, creatorId);
+
+            var isCarValid = _carService
+                                .CheckIsCarBelongsToUserByIds(createTripDTO.TransportationCarId, creatorId);
```

Comment:
I think, you should check also is car verified, because if not, user shouldn't be eligable to use it in the trip.